similarity/src/main/java/edu/illinois/cs/cogcomp/nesim/utils/DameraoLevenstein.java

```diff
@@ -0,0 +1,75 @@
+package edu.illinois.cs.cogcomp.nesim.utils;

```

Comment:
todo: try replacing this with core-utilities' Levenstein distance (https://github.com/CogComp/cogcomp-nlp/blob/master/core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/algorithms/LevensteinDistance.java)